spring-pulsar-sample-apps/src/main/java/app1/SpringPulsarBootApp.java

```diff
@@ -107,13 +109,35 @@ void listen3(Foo message) {
 		this.logger.info("Message received :" + message);
 	}
 
+	/*
+	 * Create a partitioned topic using PulsarAdministration and then publish to the topic
+	 * and consume from it.
+	 */
+	@Bean
+	ApplicationRunner runner4(PulsarTemplate<String> pulsarTemplate, PulsarAdministration pulsarAdministration) {
```

Comment:
@alpreu I think we can replace all of this w/ a `@Bean PulsarTopic` now that the auto-config is in place.